src/main/java/seedu/address/model/person/Remark.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's remarks in the address book.
+ * Guarantees: immutable; is always valid
+ */
+public class Remark {
+
+    public final String value;
+
+    public Remark(String remark) {
+        requireNonNull(remark);
+        value = remark;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Remark // instanceof handles nulls
+                && value.equals(((Remark) other).value)); // state check
```

Comment:
The indentation could be improved to make the bracket-ing clear || and && should not be on the same indentation level. :)